draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -1023,15 +1057,28 @@ Steps:
 
 ## Interface
 
-The generic interface for deterministic encoding functions to elliptic curves is as follows:
+The generic interface shared by all encodings in this section is as follows:
+
+~~~
+(x, y) = map2curve(u)
+~~~
+
+The output (x, y) specifies a point on an elliptic curve defined over base field F;
+x and y are elements of F.
+
+The input u is an element of F that MUST be the output of the hash2base function
+({{hashtobase}}). The argument ctr to hash2base SHOULD be 0, except as described
+in {{rom}}.
```

Comment:
> The argument ctr to hash2base SHOULD be 0, except as described in Section 6.

I prefer not to have by-default values.

Reply:
Sorry, I don't understand this comment.

It seems pretty clear that everyone should choose the same value of `ctr`. I'm happy to change the definition of `hash2base` so that `ctr` is an integer in {0,1} rather than in [0, 256), but barring that change there must be an agreed-upon value in order for implementations to interoperate.

Reply:
I was not clear in this comment, please disregard it. 
However, narrowing the value of the counter to {0,1} seems to me is a good idea. 
The non-RO must fix the ctr value to, for example, 0.